src/components/Pages/RequestEpisode.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect } from 'react';
-import Card from '../Cards/Card';
+import Card fromhttps://github.com/Avdhesh-Varshney/Chanakya-Niti/pull/22 '../Cards/Card';
```

Comment:
what's the purpose of these comments and links addition?

Reply:
i have removed these comments and links